### ISSUE TITLE
Change the document about twistd plugins to not redefine the word "TAP"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2396]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2396|
|Type|enhancement|
|Created|2007-01-13 16:55:58Z|

That was a stupid idea. What the crap? Just talk about "twistd plugins", please.

Attachments:

* [tap-sucks.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d04/d045236a0e03750520fa4d9cfa962fd38f11217d/7891f5f37a6cffa5d8449f18025cdbd487e85828.patch) (2974 bytes) - added by radix on 2007-01-13 17:17:23Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2396 2396
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__Core_2_6 Core-2.6
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1168707358000000 1168707358000000
changetime__1168710031000000 1168710031000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to moonfallen|
|-|-|

Ready for review in patch tap-sucks.patch.

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|moonfallen set owner to @radix|
|-|-|

Suggesting capitalizing the "p" in the <h1> title.  Also, I it's

twistd -n -f [foo].tap

Fix then check in.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [19308]) Merge patch tap-sucks.patch with minor review fixes

Author: radix
Reviewer: moonfallen
Fixes [#2396](https://github.com/twisted/twisted/issues/2396)

Get rid of the confusing redefinition of "TAP" in the twistd plugin document,
and don't use the phrase "Twisted Application Plugin".

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
